data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report6_fitdata.py

```diff
@@ -549,6 +552,53 @@
         ignore_index=True)
 df1.T
 
+# # Verify that all records with invalid level values are being dropped from sleep_level table
+#
+# DC-2606
+#
+# Queries the sandbox table for the corresponding cleaning rule (created in DC-2605) and outputs any records that are being dropped due to invalid level values. It also outputs any records in the sleep_level table that should have been dropped but were not.
+
+query = f'''
+
+WITH df1 AS (
+SELECT 
+*, 'dropped' AS dropped_status
+FROM 
+`{project_id}.{fitbit_sandbox_dataset}.{sleep_level_sandbox_table}`),
+
+df2 AS (
+SELECT 
+*,'not dropped' AS dropped_status
+FROM 
+`{project_id}.{fitbit_dataset}.sleep_level`
+WHERE 
+(
+    LOWER(level) NOT IN 
+         ('awake','light','asleep','deep','restless','wake','rem','unknown') OR level IS NULL))
```

Comment:
I would suggest this might be more meaningful if the data is condensed to a different format rather than a "get all" format.  Perhaps
```python
WITH df1 AS (
SELECT 
level
, 'dropped' AS dropped_status,
, count(*) as n_violations
FROM 
`{project_id}.{fitbit_sandbox_dataset}.{sleep_level_sandbox_table}`
group by 1, 2),
df2 AS (
SELECT 
level
, 'not dropped' AS dropped_status,
, count(*) as n_violations
FROM 
`{project_id}.{fitbit_dataset}.sleep_level`
WHERE 
(
    LOWER(level) NOT IN 
         ('awake','light','asleep','deep','restless','wake','rem','unknown') OR level IS NULL)
)
```

@molly-giddings thoughts on this change?

Reply:
@lledezma, @lrwb-aou - I love this change.  This should give us the counts of the dropped data by distinct level which may be helpful in correcting it, and also cut down on the number of records output.  Maybe also throw in an `order by n_violations` in df1.

Reply:
@lledezma let me know when you've had a chance to review the comments and I'll take another look at the PR.